client/finality-grandpa/rpc/src/finality.rs

```diff
@@ -26,13 +26,9 @@ pub struct EncodedFinalityProofs(pub sp_core::Bytes);
 
```

Comment:
Maybe it makes sense to rename `EncodedFinalityProofs` to `EncodedFinalityProof` since now it's only one and not multiple fragments.